helm-charts/charts/kube-starrocks/values.yaml

```diff
@@ -28,36 +31,31 @@ starrocksOperator:
       cpu: 500m
       memory: 200Mi
 
-
 # deploy a starrocks cluster
 starrocksCluster:
-  # Provide a name for starprocks cluster
+  # the name of starrockscluster cluster, if not set, the chart name will be used.
   name: ""
+  # the namespace of starrockscluster cluster, if not set, the release namespace will be used.
   namespace: ""
-  # annotations for starrocks cluster
+  # annotations for starrocks cluster.
```

Comment:
this annotations is used or not?

Reply:
It is used in helm chart template to add annotation for starrockscluster

Reply:
but in starrockscluster,  operator doesn't use it at all?

Reply:
operator will use it to do restart operation. for exmaple: 
```
if !be.statefulsetNeedRolloutRestart(src.Annotations, est.Annotations) && ok {
	// when restart we add `AnnotationRestart` to annotation. so we should add again when we equal the exsit statefulset and new statefulset.
	anno := rutils.Annotations{}
	anno.Add(common.KubectlRestartAnnotationKey, est.Spec.Template.Annotations[common.KubectlRestartAnnotationKey])
	new.Spec.Template.Annotations = anno
}
```

Reply:
ok, may be more examples to be added here. sounds like only some specific annotations are acceptable here?

Reply:
good suggestion! maybe, I will check it, and add comment in detail.